racket-poppler/render-tex.rkt

```diff
@@ -44,15 +44,15 @@
   (set! defaultdir
         (or defaultdir
             (case (system-type)
-              [(macosx) "/Library/TeX/texbin"]
+              [(macosx) "/usr/texbin"]
```

Comment:
Hmm - that means we need a way to figure out whether we are on El Capitan or earlier.